server/src/main/java/org/elasticsearch/index/codec/PerFieldMappingPostingFormatCodec.java

```diff
@@ -32,43 +31,25 @@
 import org.apache.lucene.index.SegmentReadState;
 import org.apache.lucene.index.SegmentWriteState;
 import org.elasticsearch.common.lucene.Lucene;
-import org.elasticsearch.index.mapper.MappedFieldType;
-import org.elasticsearch.index.mapper.MapperService;
 
 import io.crate.lucene.codec.CustomLucene90DocValuesFormat;
 import io.crate.types.FloatVectorType;
 
 
 /**
  * {@link PerFieldMappingPostingFormatCodec This postings format} is the default
- * {@link PostingsFormat} for Elasticsearch. It utilizes the
- * {@link MapperService} to lookup a {@link PostingsFormat} per field. This
- * allows users to change the low level postings format for individual fields
- * per index in real time via the mapping API. If no specific postings format is
- * configured for a specific field the default postings format is used.
+ * {@link PostingsFormat} for Crate. It disables compression on docvalues terms
+ * dictionaries, and increases the max supported vector dimension to 2048
```

Comment:
minor, maybe you can replace hard-coded 2048 with: `{@ link FloatVectorType#MAX_DIMENSIONS}`

Reply:
Have done so.  I've also renamed the whole file to CrateCodec, as we don't actually do any per-field postings format changes here anymore.